modules/machine-user-infra-machines-pxe.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
+
+[id="machine-user-infra-machines-pxe_{context}"]
+= Creating more {op-system-first} machines by PXE or iPXE booting
+
+You can create more compute machines for your bare metal cluster by using
+PXE or iPXE booting.
+
+.Prerequisites
+
+* Obtain the URL of the Ignition config file for the compute machines for your
+cluster. You uploaded this file to your HTTP server during installation.
+* Obtain the URLs of the {op-system} ISO image, compressed metal BIOS, `kernel`
+and `initramfs` files that you uploaded to your HTTP server during cluster
+installation.
+* Access to the PXE booting infrastructure that you used to create the machines
+for your {product-title} cluster during installation. The machines must boot
+from their local disks after {op-system} is installed on them.
+* If you use UEFI, you must have access to the `grub.conf` file that you modified
+during {product-title} installation.
+
+.Procedure
+
+. Confirm that your PXE or iPXE installation for the {op-system} images is
+correct.
+
+** For PXE:
++
+----
+DEFAULT pxeboot
+TIMEOUT 20
+PROMPT 0
+LABEL pxeboot
+    KERNEL http://<HTTP_server>/rhcos-<version>-<architecture>-installer-kernel <1>
+    APPEND ip=dhcp rd.neednet=1 initrd=http://<HTTP_server>/rhcos-<version>-<architecture>-installer-initramfs.img console=tty0 console=ttyS0 coreos.inst=yes coreos.inst.install_dev=sda coreos.inst.image_url=http://<HTTP_server>/rhcos-<version>-<architecture>-metal-bios.raw.gz coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <2> <3>
```

Comment:
change bootstrap.ign to worker.ign

Reply:
Just when you add more workers, or also during initialization?

Reply:
For all of these similar references, too. This method's reused from the installation flow, and I don't want to wreck that process.

Reply:
When you are running worker installation, both initial install or scaling up, you need to use worker.ign file.

bootstrap.ign is only used for the bootstrap node, in the initial installation